File: mobile/lib/models/OpenHour.dart
import 'package:mobile/models/Company.dart';

class OpenHour{
  String id;
  int dayOfWeek;
  DateTime openMorning;
  DateTime closeMorning;
  DateTime openEvening;
  DateTime closeEvening;
  DateTime openNight;
  DateTime closeNight;   
  Company company; 
}

File: backend/src/models/OpenHour.js
const mongoose = require('../config/database');

const Schema = mongoose.Schema;

const OpenHour = new Schema({
    day_week : { type: Number, required : true },
    open_morning : { type : Date, required : false },
    close_morning : { type : Date, required : false },
    open_evening : { type : Date, required : false },
    close_evening : { type : Date, required : false },
    open_night : { type : Date, required : false },
    close_night : { type : Date, required : false },
});

module.exports = mongoose.model('OpenHours', OpenHour);